<launch>
	<!-- For find_object node use calling service -->
	<!-- <arg name="image_topic" default="/to_mmdetector" /> -->
	<!-- For find_object node use subscribing -->
	<!-- <arg name="visualization" default="false" /> -->
	<arg name="mmdet_pub_rate" default="10" />

	<node pkg= "mmdetection_ros" name="cargo_detector" type="cargo_detector.py"  output="screen"> 
		<param name="~publish_rate" value="$(arg mmdet_pub_rate)" />
	</node>
</launch>
